USE_CASES.md

```diff
@@ -110,4 +110,16 @@ end
 puts response.status_code
 puts response.body
 puts response.headers
-```
\ No newline at end of file
+```
+
```

Comment:
Please add entries into the [Table of Contents](https://github.com/sendgrid/sendgrid-ruby/blob/master/USE_CASES.md#table-of-contents) and we will be good to go :)

Reply:
Nice catch! PR has been updated.